docs/source/releases/v1_11_7a0.rst

```diff
@@ -141,3 +142,19 @@ require changes to the output products as well.
     modified: tests/yaml_configs/abi_test.yaml
     modified: tests/yaml_configs/abi_test_low_memory.yaml
 
+Testing Updates
```

Comment:
I would say perhaps we *do* update recenter_tc to add a pytest for that output chekcer - so we can see how we want to tie the pytests together for various interfaces (if at all).

Reply:
Ok, I can look into that in the future. However, this will essentially be tested the same as the text image checker, so I'm not actually sure if it's necessary, considering it uses the text_checker `outputs_match` function.

Reply:
I still think we should add the test to recenter_tc, because it may inform how we set up the base test.  @jsolbrig what do you think?

Reply:
recenter_tc test added!